package.json

```diff
@@ -1,30 +1,14 @@
 {
   "name": "page-metadata-service",
-  "version": "0.1.0",
   "description": "A RESTful microservice for generating preview summaries of web pages.",
-  "main": "service.js",
-  "scripts": {
-    "test:lint": "eslint .",
-    "test": "test"
-  },
-  "repository": {
-    "type": "git",
-    "url": "git+https://github.com/mozilla/page-metadata-service.git"
-  },
-  "keywords": [
-    "page",
-    "summary",
-    "metadata",
-    "rest",
-    "service",
-    "api"
-  ],
+  "version": "0.1.0",
   "author": "Jared Kerim",
-  "license": "MPL-2.0",
+  "bin": {
+    "page-metadata-service": "bin/cli.js"
+  },
```

Comment:
Not sure if this is needed. I think it only adds the `page-metadata-service` alias if you install this module globally or locally.